src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -629,11 +700,55 @@ def _revYangConvert(val):
 
         return vValue
 
+    """
+    Rev xlate from <TABLE>_LIST to table in config DB
+    QOS MAP Yang has netsted list, each nested list key:val should 
+    be mapped to field:value in Config DB. 
+    """
+
+    def _revQosMapXlateList(self, model, yang, config, table):
+        # get keys from YANG model list itself
+        listKeys = model['key']['@value']
+        # create a dict to map each key under primary key with a dict yang model.
+        # This is done to improve performance of mapping from values of TABLEs in
+        # config DB to leaf in YANG LIST.
+        leafDict = self._createLeafDict(model, table)
+
+        # Gather netsted list key and value from model  
+        nested_clist = model.get('list')
+        if nested_clist:
+            nested_map = nested_clist['@name']
+            nested_listKey = nested_clist['key']['@value']
+            nested_leafDict = self._createLeafDict(nested_clist, table)
+            for lkey in nested_leafDict:
+                if nested_listKey != lkey:
+                    nested_listVal = lkey
+
+        # list with name <NAME>_LIST should be removed,
+        if "_LIST" in model['@name']:
+            for entry in yang:
+                # create key of config DB table
+                pkey, pkeydict = self._createKey(entry, listKeys)
+                self.sysLog(syslog.LOG_DEBUG, "revXlateList pkey:{}".format(pkey))
+                config[pkey]= dict()
+                # fill rest of the entries
+                nested_list = entry[nested_map]
+                for index in range(len(nested_list)):
```

Comment:
if we say "entry in nested_list"....then we can use entry[inner_listKey] and entry[inner_listVal]